supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -445,13 +445,25 @@ Always spell out the full product name of the host, and do not capitalize the te
 [discrete]
 [[red-hat-openshift-container-storage]]
 ==== Red Hat OpenShift Container Storage (noun)
-*Description*: Red Hat software-defined storage for containers that helps to develop and deploy applications quickly and efficiently across cloud platforms.
+*Description*: Red Hat software-defined storage for containers that helps to develop and deploy applications quickly and efficiently across cloud platforms. In 2021, Red Hat OpenShift Container Storage was rebranded as Red Hat OpenShift Data Foundation.
 
 *Use it*: yes
```

Comment:
Recommend changing this to "no", since it's an old name.

(We do this for other old name entries, like "Red Hat JBoss Data Grid")

Reply:
Good catch!